Source/Core/Core/ActionReplay.cpp

```diff
@@ -134,20 +133,22 @@ void AddCode(ARCode code)
   }
 }
 
-void LoadAndApplyCodes(const IniFile& global_ini, const IniFile& local_ini)
+void LoadAndApplyCodes(Config::Layer& global_ini, Config::Layer& local_ini)
```

Comment:
Then make it a pointer. Only const refs in argument lists, otherwise you can't tell what's being modified when a function is called.